prime-router/src/main/kotlin/cli/LivdTableDownload.kt

```diff
@@ -1,69 +1,93 @@
+import com.github.ajalt.clikt.parameters.options.required
 import it.skrape.core.htmlDocument
 import it.skrape.fetcher.HttpFetcher
 import it.skrape.fetcher.response
 import it.skrape.fetcher.skrape
 import it.skrape.selects.eachHref
 import it.skrape.selects.html5.a
+import org.apache.commons.io.FilenameUtils
+import org.apache.poi.hssf.usermodel.HSSFWorkbook
+import org.apache.poi.ss.usermodel.CellType
+import org.apache.poi.ss.usermodel.Row
+import org.apache.poi.ss.usermodel.Sheet
+import org.apache.poi.ss.usermodel.Workbook
+import org.apache.poi.xssf.usermodel.XSSFWorkbook
 import java.io.File
+import java.io.FileInputStream
 import java.io.FileOutputStream
 import java.net.URL
 
-private const val cdcLOINCTestCodeMappingPage = "https://www.cdc.gov/csels/dls/sars-cov-2-livd-codes.html"
+/**
+ * cdcLOINCTestCodeMappingPageUrl is the CDC URL that contains the LIVD-SARS-CoV-2-yyyyMMdd.xlsx file.
+ */
+private const val cdcLOINCTestCodeMappingPageUrl = "https://www.cdc.gov/csels/dls/sars-cov-2-livd-codes.html"
+
+/**
+ * livdSARSCov2File is the prefix of the LIVD-SARS-CoV-2-yyyyMMdd.xlsx file to download.
+ */
 private const val livdSARSCov2File = "LIVD-SARS-CoV-2"
 
+/**
+ * sheetName is the sheet name with the LIVID-SARS-CoV-2-yyyyMMdd.xlsx downloaded file.
+ */
+private const val sheetName = "LOINC Mapping"
+
 /**
  * LivdTableDownload is the command line interface for the livd-table-download command. It parses the command line
  * for option given as below.
  */
-class LivdTableDownload() : CliktCommand(
+class LivdTableDownload : CliktCommand(
     name = "livd-table-download",
     help = """
     livd-table downloads the latest LOINC test data, so it can be ingested automatically. 
     
-    It looks for the LIVD-SAR-CoV-2-yyyy-MM-dd.xlsx file from $cdcLOINCTestCodeMappingPage.
+    It looks for the LIVD-SAR-CoV-2-yyyy-MM-dd.xlsx file from $cdcLOINCTestCodeMappingPageUrl.
 	If the file is found, it downloads the file into the directory specified by the --output-dir <path> option.
 	If the option is not specified, it will download the file to ./build directory.
         
-    Example:
-      The following command will download the latest LIVD-SARS-CoV-2-yyyy-MM-dd.xlsx from the above URL.  It will 
-      store the file under the ./junk directory.
+    Example: The following command will download the latest LIVD-SARS-CoV-2-yyyy-MM-dd.xlsx from the above URL.  
+      It will store the LIVD-SARS-CoV-2-xyz.csv file under the ./junk directory.
       
-         ./prime livd-table --output-dir ./junk 
+         ./prime livd-table-download --output-file ./junk/LIVD-SARS-CoV-2-xyz.csv
     """
 ) {
     private val defaultOutputDir = "./build"
-    private val outputDir by option(
-        "--output-dir",
-        metavar = "<path>",
-        help = "interpret `--output` relative to this directory (default: \"$defaultOutputDir\")"
-    ).default(defaultOutputDir)
+    private val outputDirFile by option(
```

Comment:
Change the name of the variable to match its function.  Take the time when you open a PR to go over all the changes you make and make sure everything reads and makes sense.

Reply:
Done.  Thanks.